src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1183,6 +1183,68 @@ TEST_CASE("FastLSTMLayerParametersTest", "[ANNLayerTest]")
   REQUIRE(layer1.Rho() == layer2.Rho());
 }
 
+/**
+ * Check whether copying and moving network with FastLSTM is working or not.
+ */
+TEST_CASE("CheckCopyFastLSTMTest", "[ANNLayerTest]")
+{
+ arma::cube input = arma::randu(1, 1, 5);
+ arma::cube target = arma::ones(1, 1, 5);
+ const size_t rho = 5;
+
+ RNN<NegativeLogLikelihood<> > *model1 =
+    new RNN<NegativeLogLikelihood<> >(rho);
+ model1->Predictors() = input;
+ model1->Responses() = target;
+ model1->Add<Linear<> >(1, 10);
+ model1->Add<FastLSTM<> >(10, 3, rho);
+ model1->Add<LogSoftMax<> >();
+
+ ens::StandardSGD opt(0.1, 1, 5, -100, false);
+ model1->Train(input, target, opt);
+
+ arma::cube predictions1;
+ model1->Predict(input, predictions1);
+
+ RNN<> model2() = *model1;
```

Comment:
Hey @Abilityguy thanks for contributing.
Just remove `()` from here we don't need them to call copy constructor.
```
RNN<> model2 = *model1;
```